psi4/driver/driver_cbs.py

```diff
@@ -1586,6 +1583,16 @@ def _cbs_gufunc(func, total_method_name, **kwargs):
         else:
             return ptype_value
 
+    # Drop out for props and freqs
+    elif ptype == "properties":
+        raise ValidationError("Properties: Cannot extrapolate or delta correct properties yet.")
+    elif ptype == "frequency":
+        raise ValidationError("Frequency: Cannot extrapolate or delta correct frequencies yet.")
```

Comment:
No reason we cannot do this for `freqs`, should get around to testing that someday. Can we also add an `else` here at the very end of `ptype` is not matched?

Reply:
should i do:
```
else:
    pass
```
or shift everything in by one indent?